lib/Widget/Definition/Rule.php

```diff
@@ -31,6 +31,9 @@ class Rule implements \JsonSerializable
 
     public $onStatus = true;
 
+    /** @var string|null */
+    public $message;
```

Comment:
This has just been moved to the test node (on purpose). Sorry if I was unclear, but I meant "use the property on test in locale.php"